Gruntfile.js

```diff
@@ -34,154 +34,135 @@ module.exports = function (grunt) {
                 sourceDir: 'templates/twig-source',
                 sourceFiles: '<%= config.templates.sourceDir %>/**/*.html',
                 generatedDir: 'templates/generated',
-                generatedFiles: '<%= config.templates.generatedDir %>/**/*.html'
+                generatedFiles: '<%= config.templates.generatedDir %>/**/*.html',
             },
 
             lintFiles: {
                 node: ['Gruntfile.js'],
                 es: ['<%= config.srcScriptsDir %>'],
-                tests: ['<%= config.testsDir %>']
-            }
-        }
+                tests: ['<%= config.testsDir %>'],
+            },
+        },
 
     });
 
     grunt.config.merge({
         watch: {
             es: {
                 files: [
-                    '<%= config.srcScriptsDir %>/**/*.es.js',
-                    '<%= config.compiledScriptsDir %>/**/templates.js'
+                    '<%= config.srcScriptsDir %>/**/*.js',
+                    '<%= config.compiledScriptsDir %>/**/templates.js',
                 ],
                 tasks: [
                     'compilejs',
-                    'uglify'
-                ]
+                    'uglify',
+                ],
             },
             swig: {
                 files: [
-                    '<%= config.templates.sourceFiles %>'
+                    '<%= config.templates.sourceFiles %>',
                 ],
                 tasks: [
-                    'orderedSwig'
-                ]
+                    'orderedSwig',
+                ],
             },
             ngtemplates: {
                 files: ['<%= config.templates.generatedFiles %>'],
-                tasks: ['ngtemplates']
-            }
-        },
-        browserify: {
-            all: {
-                files: {
-                    '<%= config.distDir %>/language-select.js': '<%= config.compiledScriptsDir %>/**/*.js'
-                }
-            }
+                tasks: ['ngtemplates'],
+            },
         },
+        browserify: {all: {files: {'<%= config.distDir %>/language-select.js': '<%= config.compiledScriptsDir %>/**/*.js'} } },
```

Comment:
I think this is less readable inline. Grunt config lends itself well to indentation so you can easily compare settings between tasks by scanning up/down a column of indentation